script/maybe-run.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @overview Run a tool if it's present on the PATH, otherwise output a warning.
+ * @license MIT-0
+ */
+
+import process from "node:process";
+
+import which from "which";
+
+import { common } from "./_.js";
+
+if (common.argv.length === 0) {
+  console.error("Provide a command to try and execute.");
+  process.exit(1);
+}
+
+const cmd = common.argv[0];
+const args = common.argv.slice(1);
+
+try {
+  which.sync(cmd);
+} catch (_) {
+  console.warn(
+    `Command '${cmd}' not found, it will not be run.`,
+    "Install it to make this warning go away.",
+  );
+  process.exit(0);
```

Comment:
I think in a CI context this should either be an error, or the check should be foregone completely. In a CI context we intent for the tool the run, and it shouldn't fail silently if the tool is missing in CI.

Reply:
Resolved with de29baae3ab02e4135f337e187d429b10f54fb90 and dcfc6660d070f00f20cdcec13a0c39b61aee2de1